runhttpd.sh

```diff
@@ -1,11 +1,22 @@
 #!/usr/bin/bash
 
+IP=${IP:-"172.22.0.1"}
 HTTP_PORT=${HTTP_PORT:-"80"}
 
-rmdir /var/www/html
-ln -s /shared/html/ /var/www/html
+mkdir -p /shared/html
+chmod 0777 /shared/html
+
+# Copy files to shared mount
+cp /tmp/inspector.ipxe /shared/html/inspector.ipxe
+cp /tmp/dualboot.ipxe /shared/html/dualboot.ipxe
+
+# Use configured values
+sed -i -e s/IRONIC_IP/$IP/g -e s/HTTP_PORT/$HTTP_PORT/g /shared/html/inspector.ipxe
 
 sed -i 's/^Listen .*$/Listen '"$HTTP_PORT"'/' /etc/httpd/conf/httpd.conf
+sed -i -e 's|\(^[[:space:]]*\)\(DocumentRoot\)\(.*\)|\1\2 "/shared/html"|' \
+    -e 's|<Directory "/var/www/html">|<Directory "/shared/html">|' \
+    -e 's|<Directory "/var/www">|<Directory "/shared">|' /etc/httpd/conf/httpd.conf
```

Comment:
Looks good, but I don't think we should override Directory /var/www, only Directory /var/www/html.  /var/www contains ScriptAlias /var/ww/cgi-bin and we don't want to override that.

Reply:
There is no real reason to I guess since we don't have any cgi-bin content... then again for monitoring... we may end up with something.